edpop-record-ontology.ttl

```diff
@@ -0,0 +1,140 @@
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix owl: <http://www.w3.org/2002/07/owl#> .
+@prefix dc: <http://purl.org/dc/elements/1.1/> .
+@prefix dcterms: <http://purl.org/dc/terms/> .
+@prefix skos: <http://www.w3.org/2004/02/skos/core#> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+@prefix edpoprec: <https://dhstatic.hum.uu.nl/edpop-records/latest/> .
+
+edpoprec: a owl:Ontology ;
+          dcterms:license <https://creativecommons.org/licenses/by/3.0/> .
```

Comment:
As far as I can see, software and CC licenses are used. I would not consider this controlled vocab file software, but I agree that this might be a grey area. I would say the CC-BY license is fine (although I would recommend v4.0 over v3.0). 

This search query on github gives some examples of licenses being used: https://github.com/search?q=%22dcterms%3Alicense%22+path%3A*.ttl&type=code&ref=advsearch&p=1. Also not conclusive.